From 64dfefa72f2e0bcbd1012722f4b065de4c94f22d Mon Sep 17 00:00:00 2001
From: Wentao Guan <guanwentao@uniontech.com>
Date: Mon, 24 Feb 2025 17:09:32 +0800
Subject: [PATCH 356/360] deepin: config: adjust PC104 config

CONFIG_PC104 possible in x86(and it also uncommon),
before disable it x86, disable it on arm64 and loongarch.
---
 arch/arm64/configs/deepin_arm64_desktop_defconfig         | 1 -
 arch/loongarch/configs/deepin_loongarch_desktop_defconfig | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/deepin_arm64_desktop_defconfig b/arch/arm64/configs/deepin_arm64_desktop_defconfig
index 4518630d5819..970f9aa4e13f 100644
--- a/arch/arm64/configs/deepin_arm64_desktop_defconfig
+++ b/arch/arm64/configs/deepin_arm64_desktop_defconfig
@@ -43,7 +43,6 @@ CONFIG_BLK_DEV_INITRD=y
 CONFIG_EXPERT=y
 CONFIG_SGETMASK_SYSCALL=y
 CONFIG_KALLSYMS_ALL=y
-CONFIG_PC104=y
 CONFIG_PROFILING=y
 CONFIG_KEXEC=y
 CONFIG_KEXEC_FILE=y
diff --git a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
index e34938e38946..594a4a45d56a 100644
--- a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
+++ b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
@@ -52,7 +52,6 @@ CONFIG_SCHED_AUTOGROUP=y
 CONFIG_EXPERT=y
 CONFIG_SGETMASK_SYSCALL=y
 CONFIG_KALLSYMS_ALL=y
-CONFIG_PC104=y
 CONFIG_PROFILING=y
 CONFIG_KEXEC=y
 CONFIG_CRASH_DUMP=y
-- 
2.20.1

